apps/redi-talent-pool/src/pages/app/cv/CvListPage.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react'
+
+import {
+  Heading,
+  Button,
+  Modal,
+  FormInput,
+} from '@talent-connect/shared-atomic-design-components'
+import { Section, Columns, Content } from 'react-bulma-components'
+
+import { useTpjobseekerCvCreateMutation } from '../../../react-query/use-tpjobseekercv-mutation'
+import { useTpJobseekerCvQuery } from '../../../react-query/use-tpjobseekercv-query'
+
+import { LoggedIn } from '../../../components/templates'
+import { EmptySectionPlaceholder } from '../../../components/molecules/EmptySectionPlaceholder'
+import CvListItem from './CvListItem'
```

Comment:
We should align on this, but if you agree: can we drop introducing new components that use default exports, and use only named exports instead? I've been trying to gradually refactor the codebase towards using named exports only.